tmk_core/common/keycode.h

```diff
@@ -481,9 +484,9 @@ enum internal_special_keycodes {
     KC_MS_WH_LEFT,
     KC_MS_WH_RIGHT,     /* 0xFC */
     /* Mousekey accel */
-    KC_MS_ACCEL0,
-    KC_MS_ACCEL1,
-    KC_MS_ACCEL2        /* 0xFF */
+    KC_MS_ACCEL3,
+    KC_MS_DNSPED,
+    KC_MS_UPSPED        /* 0xFF */
```

Comment:
I think we'll have a problem breaking backwards compatibilty with this PR -- there are many keymaps that use these codes. My suggestion would be to start a new range for these. Not ideal but it allows us to preserve things as they are. what do you think?